tests/data/packageVersion.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace ExamplePackageVersion;
+
+// TODO: phpunit/phpunit:<50 This has to be fixed when updating to phpunit 50.x
+
+// TODO: phpunit/phpunit:<5 This has to be fixed when updating to phpunit 5.x
+// TODO: phpunit/phpunit:5.3.* This has to be fixed when updating to phpunit 5.3.*
+
+
+// TODO: not-installed/package:<5 this should error because package is not in composer.json
+
+// TODO: phpunit/phpunit:<9
+// TODO: phpunit/phpunit:<10
```

Comment:
wouldn't this test fail when using phpunit 10? :thinking:

Reply:
right, I had had the logic reversed. I think all feedback is addressed now

Reply:
and now your rule looks even closer than mine :sweat_smile: